### ISSUE TITLE
`use strict`; rule random false result

### ISSUE THREAD

**Issue by Bartvds** [palantir/tslint#69]
Something is triggering this rule in some files that have the statement:

```
module git {
    'use-strict';

}
```
